data/successors/successor side missions.txt

```diff
@@ -0,0 +1,505 @@
+			`	"From where on the map dost thou hail, captain?" Nnesa roughly gestures to an area of their map that you recognize as human space.`
+			choice
+				`	(Indicate Rutilicus.)`
+					goto rutilicus
+				`	(Indicate Sol.)`
+					goto sol
+				`	(Gesture broadly to the expanse of space depicted on the map.)`
+			`	"Oh!" says Nnesa, "Thou art a scion of the constellations, first and foremost to the stars? 'Tis fair. Thou hast a galaxy to wander, a nearly endless sky. The things that there must be to see---"`
+				goto interrupt
+			label rutilicus
+			`	"Oh!" says Nnesa, "I am not sure I know the name for that one, captain. What dost thou call thy home?"`
+			choice
+				`	"Rutilicus."`
+					goto rutilicus2
+			label rutilicus2
+			`	Nnesa asks you to repeat the word a second time, slowly working their way through your pronunciation and transcribing a short sequence of flowing letters on their map. "'Tis a pleasant name," they say.`
+				goto interrupt
+			label sol
+			`	"Oh!" says Nnesa, "Sol." They quietly sound their way through the word a couple times, trying to get it right. "Our name for it is Saaqa-Viirja. 'Twas one of the northernmost extents of our Predecessors' voyages. Ah. But thou know'st this already, for sure."`
+				goto interrupt
+			label interrupt
+			`	The two of you are interrupted by Jaasora, who needs Nnesa's help to pack the now-deflating tent back in its cart, and once that is done they indicate that they are ready to return home.`
+	on complete
+		"successor jobs" += 2
+		payment 78000
+		conversation
+			`	Nnesa and Jaasora have continued to marvel at the stars on the way back home, in addition to spending a substantial amount of time poring over their mutual collection of photos, drawings, and maps from Maspa-Mavra. "Our deepest thanks, captain," Jaasora says to you once you have landed in the spaceport as Nnesa wrestles with the tent and cart. "'Twas a true and joyful voyage." Nnesa calls them over; the tent clearly requires at least two to handle. Jaasora presents you with your payment of <payment> and goes to help, and the two of them maneuver their equipment out of your cargo bay before shouting a hasty but good-humored farewell.`
+
+mission "Successors: Graffiti"
+	minor
+	source
+		government "Successor"
+	to offer
+		has "known to the successors"
+		random < 15
+	on offer
+		conversation
+			`Shortly after you disembark the <ship> for the spaceport on <source>, you hear a quiet but noticeable clattering from behind you near where your ship is being refueled.`
```

Comment:
Should this be a `landing` mission?

Reply:
I figured that First Last may not necessarily disembark their ship every time they land at a spaceport. This mission relies on FL disembarking (either to notice and interact with the "repair crew" or to give the "crew" sufficient time to graffiti the ship), so I think keeping it as a spaceport mission is more appropriate under that reasoning.